src/zeek.bif

```diff
@@ -3797,6 +3797,31 @@ function get_current_packet_header%(%) : raw_pkt_hdr
 	return std::move(hdr);
 	%}
 
+## Returns the currently processed PCAP packet's timestamp.
+##
+## Returns: The currently processed packet's timestamp.
+##
+## .. zeek:see:: get_current_packet get_current_packet_header
+##
+## .. note::
+##
+##      For ``get_current_packet_ts()`` the same limitations as for
+##      :zeek:see:`get_current_packet` apply. In particular, the return value
+##      should be considered undefined when called within event handlers raised
+##      via :zeek:see:`event`, :zeek:see:`schedule` or by recipient of Broker
```

Comment:
Hmm, yet, the user added is in `check_stats()` - that's a scheduled event, and the baselines verify the output of 0.0.

---

After looking a bit: `get_current_packet()` returns `pcap_packet` including timestamp information that should allow to build the timestamp in script land without a new BiF? I forget if there was another reason to have a separate BiF?

```

```

I think this comment could be relaxed, actually:

```
 ##      The return value of ``get_current_packet()`` further should be considered
 ##      undefined when called within event handlers raised via :zeek:see:`event`,
 ##      :zeek:see:`schedule` or by recipient of Broker messages.
```
`Packet::ToVal()` actually returns a zero-initialized structure if there wasn't a packet, I had missed that.

Reply:
Alternative proposal in DM was to add a BiF `get_packet_lag()` that handles it all internally.